### DISCUSSION TITLE
Can Typst read from the standard input  under unix?

### DISCUSSION THREAD

**Discussion by PEWOLO** [typst/typst#3334] [Q&A]
Hey everyone,

Is there any way Typst would accept input from the standard input under Unix/Linux?

**Comment by frozolotl** [comment]
Noting that it was already answered on Discord by @PgBiel[^1].
> you'd need to use `/dev/stdin` and `/dev/stdout` as files, otherwise no, it's not supported by default currently
> relevant issue is <https://github.com/typst/typst/issues/410>

[^1]: https://discord.com/channels/1054443721975922748/1088371919725793360/1203396675943530526
